nixos/modules/virtualisation/proxmox-image.nix

```diff
@@ -168,6 +185,7 @@ with lib;
     system.build.VMA = import ../../lib/make-disk-image.nix {
       name = "proxmox-${cfg.filenameSuffix}";
       inherit partitionTableType;
+      additionalSpace = config.proxmox.additionalDiskSpace;
```

Comment:
How about naming the configuration option `additionalSpace` and then do `inherit partitionTableType additionalSpace;` ?